api/src/paths/project/{projectId}/update.ts

```diff
@@ -432,7 +433,7 @@ export interface IUpdateProject {
   coordinator: object | null;
   project: object | null;
   objectives: object | null;
-  location: object | null;
+  location: { geometry: Feature[] } | null;
```

Comment:
Doesn't this also include the location description field? Unless that is somewhere else

Reply:
yes it does, updated